assets/css/neo.css

```
@@ -211,41 +211,54 @@ html[data-reduce-motion="true"] *{animation:none !important; transition:none !im
+  color:var(--amber);border:1px solid color-mix(in srgb,var(--amber) 50%,transparent);
+  box-shadow:0 0 10px -3px color-mix(in srgb,var(--amber) 50%,transparent)}
+
+/* Meta line (language / stars / forks / date) — pill chips with accent dot */
+.repo-card-meta{display:flex;flex-wrap:wrap;gap:.5rem;font-size:.72rem;color:var(--faint);margin-top:auto;align-items:center}
+.repo-card-meta span{display:inline-flex;align-items:center;gap:.3rem;
+  background:color-mix(in srgb,var(--panel-2) 80%,transparent);padding:.14rem .5rem;border-radius:8px;
+  border:1px solid var(--line)}
+.repo-card-meta span[title="звёзды"]{color:var(--amber);border-color:color-mix(in srgb,var(--amber) 35%,transparent)}
+.repo-card-meta span[title="форки"]{color:var(--accent-2);border-color:color-mix(in srgb,var(--accent-2) 32%,transparent)}
+.repo-lang-dot{display:inline-block;width:9px;height:9px;border-radius:50%;margin-right:.1rem;
+  box-shadow:0 0 9px 0 currentColor}
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)